src/tests/ftest/util/dfuse_utils.py

```diff
@@ -122,7 +122,14 @@ def check_mount_state(self, nodes=None):
                 if retcode == 0:
                     check_mounted.add(host)
                 else:
-                    state["nodirectory"].add(host)
+                    command = "cat /proc/mounts | grep dfuse"
+                    retcodes = pcmd([host], command, expect_rc=None)
+                    for ret_code, nodes in list(retcodes.items()):
```

Comment:
(pylint-redefined-argument-from-local) Redefining argument with the local name 'nodes'